autospec/buildreq.py

```diff
@@ -308,6 +308,35 @@ def rakefile(filename):
                 print("Rakefile-new: rubygem-" + s)
 
 
+def parse_cmake(filename):
+    """
+    Scan a .cmake or CMakeLists.txt file for what's it's actually looking for
+    """
+    findpackage = re.compile(r"find_package\((\w+)\b.*\)", re.I)
+    pkgconfig = re.compile(r"pkg_check_modules\(\w+ (.*)\)", re.I)
+    pkg_search_modifiers = {'REQUIRED', 'QUIET', 'NO_CMAKE_PATH',
+                            'NO_CMAKE_ENVIRONMENT_PATH', 'IMPORTED_TARGET'}
+
+    with open(filename, "r", encoding="latin-1") as f:
+        lines = f.readlines()
+    for line in lines:
+        match = findpackage.search(line)
+        if match:
+            module = match.group(1)
+            try:
+                pkg = config.cmake_modules[module]
+                add_buildreq(pkg)
+            except:
+                pass
+
+        match = pkgconfig.search(line)
+        if match:
+            for s in re.split(r'\s+', match.group(1)):
+                if s in pkg_search_modifiers:
+                    continue
+                add_pkgconfig_buildreq(s)
```

Comment:
This is adding deps like `pkgconfig("systemd")` (with the quotes). Investigating.